relayer/spy_relayer/src/listener/validation.ts

```diff
@@ -230,9 +28,22 @@ export type ParsedVaa<T> = {
 
 export type ParsedTransferPayload = {
   amount: BigInt;
-  originAddress: Uint8Array; //hex
+  originAddress: string; // hex
   originChain: ChainId;
-  targetAddress: Uint8Array; //hex
+  targetAddress: string; // hex
```

Comment:
I changed these to make this type match this parse function perfectly:

https://github.com/certusone/wormhole/blob/3940c95bfb97f7866df92837d6b2f05b9591a24f/sdk/js/src/utils/parseVaa.ts#L55-L62

The `ParseTransferPayload` type is only within the relayer codebase and is not used anywhere else.